azkaban-web-server/src/main/java/azkaban/imagemgmt/servlets/ImageTypeServlet.java

```diff
@@ -74,10 +82,36 @@ protected void handleGet(
       throws ServletException, IOException {
     try {
       final Map<String, String> templateVariableToValue = new HashMap<>();
-      if (IMAGE_TYPE_WITH_ID_URI_TEMPLATE.match(req.getRequestURI(), templateVariableToValue)) {
+      ImageTypeDTO imageTypeDTO = null;
+      if (StringUtils.isNotBlank(req.getPathInfo())) {
```

Comment:
Will there be a scenario when the pathInfo can be whitespaces? if not, then we do not need to use this API and instead use String's isEmpty API.

Reply:
Good Point! Thank you!